src/actions/account.js

```diff
@@ -304,10 +303,7 @@ export const {
         wallet.setupRecoveryMessage.bind(wallet),
         () => showAlert()
     ],
-    SETUP_RECOVERY_MESSAGE_NEW_ACCOUNT: [
-        wallet.setupRecoveryMessageNewAccount.bind(wallet),
-        () => showAlert({ onlyError: true })
-    ],
+    SETUP_RECOVERY_MESSAGE_NEW_ACCOUNT: wallet.setupRecoveryMessageNewAccount.bind(wallet),
```

Comment:
It can be also solved with `globalAlertPreventClear` added to `push` state, like here: 
https://github.com/near/near-wallet/blob/master/src/actions/account.js#L149

Reply:
Correct me if I'm wrong, but I think that `globalEventPreventClear` is actually the inverse of what this change accomplishes -- it _prevents_ an alert from being cleared when we do a router redirect to a new route, which is actually the opposite of what this change did, since we don't want the alert to be displayed _at all_ in this case.  

Also, the problem is a lower level one across all of the actions that have statically bound showAlert()s... sometimes, we may not be doing a redirect immediately after dispatching an action that has a static `showAlert()` bound to it; sometimes we will just be re-rendering the same screen but don't want an alert to be thrown, and other times we may be kicking off another action based on the initial one, but with different behavior based on whether or not the previous one was thrown, for example, this happens in this PR inside of wallet.js, when we automatically fallback to implicit account creation when trying to create a funded account fails due to no funds being available in the  source account.  The key thing is that we don't want any alert shown, _unless_ the calling code that dispatched the action _wants_ to display one using e.g. showCustomAlert()

Reply:
Right, from the description I understood that you want to show an alert after redirection, that the problem is that alert is cleared while redirecting and disappears.

Reply:
Gotcha -- no, in this case it's that we don't want the alert at all, and actually we just redirect the user to the implicit flow -- e.g. the error thrown by the action is handled by the caller, so it's _really_ an error -- so the rrror shouldn't pop up at all, either before _or_ after redirecting :)